docs/quickstart.mdx

```diff
@@ -0,0 +1,209 @@
+---
+sidebar_position: 0
+sidebar_label: Quickstart
+description: "Create your first supergraph API with Hasura DDN in less than a minute."
+keywords:
+  - hasura ddn
+  - graphql api
+  - quickstart
+  - getting started
+  - guide
+  - quickstart
+hide_table_of_contents: true
+is_guide: true
+---
+
+import Step from "@site/src/components/CodeStep";
+import InstallTheCli from "@site/docs/_install-the-cli.mdx";
+import CodeBlock from "@theme/CodeBlock";
+import SimpleVideo from "@site/src/components/SimpleVideo";
+import Admonition from "@theme/Admonition";
+
+# Quickstart with Hasura DDN
+
+<div className="flex flex-col gap-6 mb-10">
+  <div className="">
+    {/* If you break this onto another line it will lose its styling because the markdown rendering engine will add a <p> tag. Please leave it as a single line. Thanks! */}
+    <p className="text-[1.3rem]">
+      In less than <em>a minute</em> and without needing a data source connection string, you can have a supergraph API
+      running locally and deployed on Hasura DDN. 🚀{" "}
+    </p>
+  </div>
+  <div className="max-w-[750px]">
+    <SimpleVideo videoLink="https://www.youtube.com/embed/OsO6TzwFb30" />
```

Comment:
Discussion: What if we just had this as a link: "Watch the video speedrun."

Reply:
I'm a fan of this 👍